lib/command/Command.js

```diff
@@ -164,73 +164,52 @@ class Command {
     }
 
     permissionCheck(msg) {
-        let req = false;
-        if(this.requirements.custom && typeof this.requirements.custom === "function") {
-            req = true;
-            if(this.requirements.custom(msg)) {
-                return true;
-            }
+        if(this.requirements.custom && typeof this.requirements.custom === "function" && !this.requirements.custom(msg)) {
```

Comment:
Why not add `await` to the function calls to add support for async functions? Would require a docs update. This also applies to a bunch of other things that can be functions rather than plain values. Not required, just something to consider.

Reply:
working on it